internal/broker/poller.go

```diff
@@ -43,3 +43,18 @@ func (p *PassthroughPoller) Invoke(logic func() (bool, error)) error {
 	}
 	return err
 }
+
+type TimerPoller struct {
+	PollInterval time.Duration
+	PollTimeout  time.Duration
+	Log func(args ...any)
+}
+
+func (p *TimerPoller) Invoke(logic func() (bool, error)) error {
+	var start = time.Now()
+	result := wait.PollImmediate(p.PollInterval, p.PollTimeout, func() (bool, error) {
```

Comment:
`pollImmediate` is deprecated in latest Go version.

Reply:
We decided not to do it right now.